packages/core/src/components/buttons/buttons.module.scss

```diff
@@ -6,6 +6,10 @@
 	cursor: pointer;
 }
 
+.button + .button {
+	margin-left: 15px;
```

Comment:
probably we can use the value from the `theming/variables` here
`margin-left: $space-3;`